.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Describe the bug**
```

Comment:
Probably, but I think this is more fool proof and I don't mind if the result reads more like a dialogue rather than a document.

Do you think it is very bad to keep it this way?